### PR TITLE
Update support site string for FAQ page

### PR DESCRIPTION
**Pull request by maxxcrawford**
Blocks https://github.com/mozilla/fx-private-relay/pull/1353

Screenshot: 

image

### CODE REVIEW

**Comment by Vinnl** [comment]
Unfortunately there were more content refinements for the FAQ; I added b0fa5e1 here to avoid multiple notifications for the translators. They'll be used in https://github.com/mozilla/fx-private-relay/pull/1356.

File: en/app.ftl [hunk 1]
```diff
@@ -250,12 +257,16 @@ faq-question-availability-answer = Free { -brand-name-relay } is available in mo
...
+faq-question-4-answer-v3 = { -brand-name-relay-premium } users can reply to a forwarded email within 3 months of receiving the email. If you add a CC or BCC when you reply back to an email, your original email address will be exposed to the recipient and those on the email. If you do not want your original email address exposed, do not add CCs or BCCs when replying.
 faq-question-4-answer-html = { -brand-name-relay } does not yet offer the ability to reply using an alias.
     If you try, nothing will happen. We are planning an additional feature to let you <a href="{ $url }" { $attrs }>reply anonymously to senders</a>.
 
 faq-question-subdomain-characters-question = What characters can I use to create a subdomain?
+# Deprecated
 faq-question-subdomain-characters-answer = Only lower case English alphabet, numbers, and dashes (-).
+faq-question-subdomain-characters-answer-v2 = You can only use lower case English alphabet, numbers, and hyphens to create a subdomain.
```
Comment (peiying2) [inline en/app.ftl:269]: Shall we use Latin alphabet instead? When we say English alphabet, we mean 26 letters but nothing outside of the list?

File: en/app.ftl [hunk 2]
```diff
@@ -214,7 +214,7 @@ faq-question-missing-emails-answer-reason-turned-off = The alias might have forw
 faq-question-missing-emails-answer-reason-delay = { -brand-name-relay } might be taking longer than usual to forward your messages
 #   $url (url) - link to the support site
 #   $attrs (string) - specific attributes added to external links
-faq-question-missing-emails-answer-b-html = If you’re a { -brand-name-relay-premium } user struggling with any of these issues, please <a href="{ $url }" { $attrs }>contact our support team</a>.
+faq-question-missing-emails-answer-support-site-html = If you’re struggling with any of these issues, please <a href="{ $url }" { $attrs }>review our support site</a>.
```
Comment (peiying2) [inline en/app.ftl:217]: Should we say `visit` our support site instead of `review` our support site?

**Review by peiying2 — COMMENTED** [review]
@maxxcrawford @Vinnl a couple of comments to check, nothing major.

File: en/app.ftl (hunk 1 above)
Comment (Vinnl) [reply en/app.ftl:269]: Hmm, I think so, but let me verify with content (also for your other remark).

File: en/app.ftl (hunk 2 above)
Comment (Vinnl) [reply en/app.ftl:217]: Btw @maxxcrawford, keep in mind that changing the key means the site will show ??? when this is merged until the PR that points to the new key is merged too.

Comment (maxxcrawford) [reply en/app.ftl]: We are using "visit". I also restored the previous string.

File: en/app.ftl (hunk 1 above)
Comment (maxxcrawford) [reply en/app.ftl]: New language is: 
> You can only use lower-case English letters, numbers, and hyphens to create a subdomain.

**Review by Vinnl — APPROVED** [review]
The suggestions were reviewed and further refined by content, so looks good to me now.

File: en/app.ftl [hunk 3]
```diff
@@ -214,7 +214,10 @@ faq-question-missing-emails-answer-reason-turned-off = The alias might have forw
 faq-question-missing-emails-answer-reason-delay = { -brand-name-relay } might be taking longer than usual to forward your messages
 #   $url (url) - link to the support site
 #   $attrs (string) - specific attributes added to external links
-faq-question-missing-emails-answer-b-html = If you’re a { -brand-name-relay-premium } user struggling with any of these issues, please <a href="{ $url }" { $attrs }>contact our support team</a>.
+faq-question-missing-emails-answer-b-html = If you’re a { -brand-name-relay-premium } user struggling with any of these issues, please <a href="{ $url }" { $attrs }>contact our support team</a>.#   $url (url) - link to the support site
```
Comment (peiying2) [inline en/app.ftl:217]: Oops, the comment at the end of the sentence needs to start a new line

**Review by peiying2 — CHANGES_REQUESTED** [review]
Thank you for revising the strings. Looks good overall.

One comment somehow sneaked into the string. Double check on that.

File: en/app.ftl (hunk 3 above)
Comment (maxxcrawford) [reply en/app.ftl:217]: Ahh! Thank you. :)

**Review by peiying2 — APPROVED** [review]
LGTM!

**Pull request merged**